rotkehlchen/tests/unit/test_protocol_balances.py

```diff
@@ -956,3 +961,76 @@ def test_extrafi_lending_balances(
             ),
         }),
     }
+
+    with globaldb.conn.read_ctx() as cursor:
+        assert cursor.execute(
+            'SELECT key, value FROM unique_cache WHERE key LIKE "EXTRAFI%"',
+        ).fetchall() == [(
+            'EXTRAFI_LENDING_RESERVES1035',
+            '0x9560e827aF36c94D2Ac33a39bCE1Fe78631088Db',
+        )]
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('network_mocking', [False])
+@pytest.mark.parametrize('base_manager_connect_at_start', [(WeightedNode(
```

Comment:
why do this?

Reply:
I used it while testing to not consume credits from the etherscan key and then I left it.